roles/openshift_logging_elasticsearch/tasks/main.yaml

```diff
@@ -229,7 +229,7 @@
       dest: "{{ tempdir }}/templates/logging-es-pvc.yml"
     vars:
       obj_name: "{{ openshift_logging_elasticsearch_pvc_name }}"
-      size: "{{ openshift_logging_elasticsearch_pvc_size }}"
+      size: "{{ (openshift_logging_elasticsearch_pvc_size | trim | length == 0) | ternary('10Gi', openshift_logging_elasticsearch_pvc_size) }}"
```

Comment:
can we use some python magic and do something like `( openshift_logging_elasticsearch_pvc_size | bool ) | ternary(openshift_logging_elasticsearch_pvc_size, '10Gi')`  if its empty it should come back as false...